.gitignore

```diff
@@ -103,3 +103,4 @@ venv.bak/
 
 # mypy
 .mypy_cache/
+/Users/zhengyuecheng/Desktop/personal/github/map-of-the-web/map-of-the-web/motw/motw/settings_secret.py
```

Comment:
This ignore file needs to be made relative -- right now it is a path on your own computer (which will not mean anything for other contributors)